specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/virtualNetwork.json

```diff
@@ -1594,6 +1597,11 @@
           "$ref": "./network.json#/definitions/SubResource",
           "default": null,
           "description": "The DDoS protection plan associated with the virtual network."
+        },
+        "bgpCommunities": {
```

Comment:
This is a breaking change and requires a new API version. See 'New property added to response' for details.
https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo
Otherwise, this change looks good

Reply:
2019-08-01 is the new API version

Reply:
You are changing 2019-08-01, which is generally not allowed for breaking changes. Ryan means introducing a newer API version.

---
In reply to: [325897421](https://github.com/Azure/azure-rest-api-specs/pull/7248#discussion_r325897421) [](ancestors = 325897421)

Reply:
It's not in `master` yet, we created a separate branch to add 2019-08-01 a couple days ago. This is PR into that branch

Reply:
Yes, you're right. Makes sense.

---
In reply to: [325919546](https://github.com/Azure/azure-rest-api-specs/pull/7248#discussion_r325919546) [](ancestors = 325919546)